.github/workflows/docker.yml

```diff
@@ -23,7 +25,7 @@ jobs:
     - uses: actions/checkout@01aecccf739ca6ff86c0539fbc67a7a5007bbc81
     - uses: elgohr/Publish-Docker-Github-Action@f7aca2fea76a5218f3c76cd5933c3ba1d8008774
       with:
-        name: gibiansky/ihaskell
+        name: daviddavo/ihaskell
```

Comment:
I don't think this belongs in this PR, I would suggest making these changes on a separate branch.